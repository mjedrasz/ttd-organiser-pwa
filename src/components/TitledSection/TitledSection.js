import React from 'react';
import Typography from '@material-ui/core/Typography';
import Paper from '@material-ui/core/Paper';
import useStyles from './styles';

export default ({ title, children }) => {

  const classes = useStyles();

  return (
    <Paper className={classes.section}>
      <Typography variant="h6" gutterBottom>
        {title}
      </Typography>
      {children}
    </Paper>
  );
};
