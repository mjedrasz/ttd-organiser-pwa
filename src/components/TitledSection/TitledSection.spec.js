import React from 'react';
import renderer from 'react-test-renderer';
import TitledSection from './TitledSection';
import { render } from '@testing-library/react';
import I18n from '@aws-amplify/core/lib/I18n';

describe('TitledSection', () => {

  test('snapshot renders correctly', () => {

    const component = renderer.create(<TitledSection title='test title'><h1>test children</h1></TitledSection>);

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});