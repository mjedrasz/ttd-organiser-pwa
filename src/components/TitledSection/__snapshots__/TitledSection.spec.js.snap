// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitledSection snapshot renders correctly 1`] = `
<div
  className="MuiPaper-root MuiPaper-elevation1 makeStyles-section-7 MuiPaper-rounded"
>
  <h6
    className="MuiTypography-root MuiTypography-h6 MuiTypography-gutterBottom"
  >
    test title
  </h6>
  <h1>
    test children
  </h1>
</div>
`;
